call-notes/call_10.md

```diff
@@ -0,0 +1,198 @@
+# Core Community Call 10 Notes
+
+Meeting Date/Time: Friday, October 20th, 2023 19:00 UTC
+
+Meeting Duration: 22 minutes 29 seconds
+
+[Video of the meeting](https://www.youtube.com/watch?v=DOaQ_VBFyIw) 
+
+## Speakers
+
+1. [Jacob Creech](https://github.com/jacobcreech)
+2. [Joe C](https://github.com/buffalojoec)
+3. [Liam Heeger](https://github.com/Iheeger-jump)
+4. [Tyera Eulberg](https://github.com/CriesofCarrots)
+5. [Galactus | Mango Market](https://github.com/godmodegalactus)
+6. Josh Siegel
+
+## Meeting Notes
+
+### Introduction
+
+Video| [00:00](https://youtu.be/DOaQ_VBFyIw?t=1)
+-|-
+
+**Jacob Creech:** Alright, welcome everyone to this month's core Community call, today we will be having Joe present on, I believe it's the feature gate threshold activation.
+
+Go ahead and take it away Joe. 
+
+### Feature Gate Threshold Automation
+
+Video |  [00:18](https://youtu.be/DOaQ_VBFyIw?t=18)
+-|-
+
+**Joe C:** Cool sounds good thanks Jacob. Hey guys I'm Joe for those of you guys who haven't met me. Yeah, I just wanted to talk a little bit about the SIMD, if any of you guys haven't seen it, it's number 72 [[SIMD 0072](https://github.com/solana-foundation/solana-improvement-documents/pull/72)]. Should I just share my screen, What do you think Jacob or ?
+
+**Jacob Creech:** Go for it.  
+
+#### Overview
+
+Video |  [00:35](https://youtu.be/DOaQ_VBFyIw?t=35)
+-|-
+
+**Joe C:** Yeah, I am not going to go through every crazy nit bit detail but we can talk a little bit about the key point and stuff, whatever you guys kind of want to jam on but basically, this is the proposal [here](https://github.com/buffalojoec/solana-improvement-documents/simd-feature-gate-threshold-automation/proposals/0072-feature-gate-threshold-automation.md). If someone coud please provide the link, that'd be great.

```

Comment:
```suggestion
**Joe C:** Yeah, I am not going to go through every crazy nit bit detail but we can talk a little bit about the key point and stuff, whatever you guys kind of want to jam on but basically, this is the proposal [here](https://github.com/buffalojoec/solana-improvement-documents/simd-feature-gate-threshold-automation/proposals/0072-feature-gate-threshold-automation.md). If someone could please provide the link, that'd be great.

```

Reply:
Alright I'd go through and correct all spelling errors.